src/Moq/ISetup.cs

```diff
@@ -24,5 +24,25 @@ public interface ISetup
 		///   Gets whether this setup is disabled.
 		/// </summary>
 		bool IsDisabled { get; }
+
+		/// <summary>
+		///   Gets whether this setup is verifiable.
+		/// </summary>
+		bool IsVerifiable { get; }
+
+		/// <summary>
+		///   Verifies this setup.
+		///   That is, checks whether it has been matched by at least one invocation on the mock.
```

Comment:
For future reference, this is actually slightly inaccurate: an `AutoImplementedProperty[GS]etterSetup` can pass verification even if it has never been invoked.